packages/react/src/themes/base/components/Slider/sliderStyles.ts

```diff
@@ -0,0 +1,92 @@
+import { SliderVariables } from './sliderVariables'
+import { SliderProps, SliderState } from '../../../../components/Slider/Slider'
+import { ComponentSlotStylesInput, ICSSInJSStyle } from '../../../types'
+import { selectors } from '../../../../styles/selectors'
+
+const getSliderSlotStyles = (p: SliderProps, v: SliderVariables, slot: 'rail' | 'track') => ({
```

Comment:
_nit_: We can simplify this if we will have:

```jsx
const getCommonSlotStyles = (p: SliderProps, v: SliderVariables) => ({
  cursor: 'pointer',
  pointerEvents: 'none',
  position: 'absolute',
  border: 0,
  height: v.railHeight,
  marginTop: `calc(${v.height} / 2 - ${v.railHeight} / 2)`,
})
```

And conditions below will go back to matching slots:
```jsx
  ...(slot === 'rail' && { width: '100%', background: v.railColor }),
  ...(slot === 'track' && { background: v.trackColor }),
  ...(p.disabled && slot === 'rail' && { background: v.disabledRailColor }),
  ...(p.disabled && slot === 'track' && { background: v.disabledTrackColor }),
```

Reply:
I like this, done, thanks